### PR TITLE
Add support for uploading documents to auto established claims

### PR DESCRIPTION
**Pull request by ATeal**
## Description of change
In order to more fully represent the 526 claim submission process we need the ability accept uploads to eventually go off to EVSS. This is part one of that. Capturing the uploads.

resolves https://github.com/department-of-veterans-affairs/vets-contrib/issues/1427

## Testing done
- [x] Rspec

## Acceptance Criteria (Definition of Done)
- can hit the endpoint and upload documents

#### Unique to this PR
- will need to make devops code updates to support the uploader credentials and make buckets in s3

#### Applies to all PRs

- [x] Appropriate logging
- [x] Swagger docs have been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [x] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: modules/claims_api/db/migrate/20190306160555_create_supporting_documents.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateSupportingDocuments < ActiveRecord::Migration
```
Comment (bastosmichael) [inline modules/claims_api/db/migrate/20190306160555_create_supporting_documents.rb]: I thought protocol requires us to put this in a separate PR?

File: modules/claims_api/config/routes.rb [hunk 2]
```diff
@@ -8,6 +8,7 @@
     namespace :forms do
       ## 526 Forms
       post '526', to: 'disability_compensation#submit_form_526'
+      post '526/:id/upload_documents', to: 'disability_compensation#upload_supporting_documents'
```
Comment (bastosmichael) [inline modules/claims_api/config/routes.rb]: I'm thinking form/526/:id/attachments might be a cleaner end point for this from a documentation standpoint but I could be wrong.

Comment (ATeal) [reply modules/claims_api/config/routes.rb]: Yeah you right... will refactor with better nomenclature

File: modules/claims_api/db/migrate/20190306160555_create_supporting_documents.rb (hunk 1 above)
Comment (ATeal) [reply modules/claims_api/db/migrate/20190306160555_create_supporting_documents.rb]: it is, I'll split it out once this pr is approved
